code-challenges.test.js

```diff
@@ -53,4 +78,10 @@ const studyMinutesWeek2 = {
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+function someValue(obj) {
+  const values = Object.values(obj)
+  values.sort((a, b) => a - b)
+  return values
+}
+
+// Pseudo code: used chatgbt to created a variable to take on all numbers in the array. using the .sort method on the values array and using the (a, b) => a - b i will sort the values from the smallest to largest. 
```

Comment:
Chat GPT is a great tool and we want our students to use all the tools available. But remember to be judicial about any result from AI. You will often have to refactor your code.